test/1_stdlib/POSIX.swift

```diff
@@ -78,6 +81,112 @@ POSIXSemaphore.test("sem_open existing O_EXCL fail") {
   let res2 = sem_unlink(semaphoreName)
   expectEqual(0, res2)
 }
+	
+// Fail because file doesn't exist.
+POSIXTests.test("fcntl fail") {
+  let fd = open(strdup(fn), 0)
+  expectEqual(-1, fd)
+  expectEqual(ENOENT, errno)
+	
+  let _ = fcntl(fd, F_GETFL)
+  expectEqual(EBADF, errno)
+}
+
+// Change modes on existing file.
+POSIXTests.test("fcntl F_GETFL/F_SETFL success with file") {
```

Comment:
Looks like each test is for one of the `fcntl()` signatures, right?  If so, could you put the signature into the test name to emphasize this?

e.g., `POSIXTests.test("fcntl(CInt, CInt): F_GETFL/F_SETFL success with file")`